class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashMap<Integer,Integer> isSeen = new HashMap<>();

        for(int i = 0; i < nums.length; i++){
            if(isSeen.containsKey(nums[i])){
                return true;
            }else{
                isSeen.put(nums[i], 1);
            }
        }
        return false;
    }
}